ddocs/medic/views/tasks_by_state/map.js

```diff
@@ -0,0 +1,6 @@
+function(doc) {
+  if (doc.type === 'task') {
+    var emission = doc.emission || {};
+    emit(doc.state, { dueDate: emission.dueDate, startDate: emission.startDate, endDate: emission.endDate });
```

Comment:
The value has 3 different dates but you're only using the `endDate` - unless you have an idea of an upcoming change using the other two I think we should only include what we need for now, eg: 

```suggestion
    emit(doc.state, doc.emission && doc.emission.endDate);
```